src/remembered-redis.ts

```diff
@@ -57,7 +62,12 @@ function prepareConfig(config: RememberedRedisConfig) {
 
 const MAX_ALTERNATIVE_KEY_SIZE = 50;
 const TTL_MAP_POS = 2;
+const DEFAULT_RETRY_COUNT = 1000;
+const DEFAULT_RETRY_DELAY = 100;
+
 export class RememberedRedis extends Remembered {
+	private static semaphoreLib: 'redlock' | 'redis-semaphore' | undefined;
```

Comment:
I think we could use an enum here for semaphore library, to avoid too much strings with `redlock` and `redis-semaphore` on this code.